VideoRenderer/VideoRenderer/SystemPlayerObserver.swift

```diff
@@ -201,12 +201,18 @@ public final class SystemPlayerObserver: NSObject {
             guard let new: AVAsset = newValue() else { return }
             emit(.didChangeAsset(new))
             
+            guard new.duration == kCMTimeIndefinite else {
```

Comment:
What is the purpose of this check?

Reply:
Try to load duration if it is indefinite

Reply:
Ok, got it.